cocotb/share/lib/vhpi/VhpiImpl.cpp

```diff
@@ -92,13 +92,23 @@ void VhpiImpl::get_sim_time(uint32_t *high, uint32_t *low)
     *low = vhpi_time_s.low;
 }
 
+template <typename T>
+int log10int(T v)
```

Comment:
```suggestion
static int log10int(T v)
```

Also, let's not template unnecessarily here. We call this in exactly one place, so let's just use `uint64_t`.

If we were templating this, it ought to be the stricter:
```C++
template<typename T, typename=std::enable_if<std::is_integral<T>::value>::type>
```